style.css

```diff
@@ -0,0 +1,91 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap");
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-family: sans-serif;
+  font-weight: 400px;
+  background: #e5e5e5;
+}
+
+section {
+  width: 375px;
+  height: 718px;
+  background: #fff;
+}

```

Comment:
As requested by the previous reviewer, kindly add the background image shown on the Figma design:
![image](https://user-images.githubusercontent.com/42799579/198427462-7787e721-7030-41e6-b3ef-70ac6a2fedba.png)

To do so, you can download it directly on [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122), then create an `images` folder inside your project and reference its URL with something like "./images/background-main-section" 😉. So, you could have something similar to this:

```css
section {
width: 375px;
height: 718px;
background: #fff;
background-image: url("./images/background-main-section")
}
```

If you are interested, you can find more information about background images [here](https://www.w3schools.com/cssref/pr_background-image.php).

Reply:
Thanks a lot both of you for the insightful review.